Source/WebCore/testing/MockPaymentCoordinator.cpp

```diff
@@ -101,7 +101,9 @@ MockPaymentCoordinator::~MockPaymentCoordinator()
 
 void MockPaymentCoordinator::dispatchIfShowing(Function<void()>&& function)
 {
-    ASSERT(showCount > hideCount);
+    if (showCount <= hideCount)
+        return;
```

Comment:
i would think that trying to do something after it's been hidden would be a bug, no?

or is this now erroneous given the above changes?

Reply:
But nothing prevents the tests from calling it after the session has been aborted, via `internals.mockPaymentCoordinator.cancelPayment();`